tmt/steps/provision/__init__.py

```diff
@@ -618,6 +619,7 @@ def ansible(self, playbook: str, extra_args: Optional[str] = None) -> None:
     def execute(self,
                 command: Union[str, List[str]],
                 test_session: bool = False,
+                log: Optional[BaseLoggerFnType] = None,
```

Comment:
Here we are enabling the verbose logging for `GuestSsh` only. I believe we should implement this consistently for all provision methods as it could be confusing to get a different behaviour for `container` or `local` provision.

Reply:
I have updated `execute()` in `podman` and `local` accordingly.

Reply:
Please, update also `Guest.execute` method, to match this signature.

I'm not sure why is no linter complaining about `GuestSsh.execute()` signature being different than that of its superclass, that should definitely be reported by mypy, at least. I find that a bit puzzling :/

Reply:
@happz Done.